build.gradle

```diff
@@ -1,11 +1,10 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
     ext {
-        compose_compiler_version = '1.4.2'
-        kotlin_version = '1.8.10'
-        compose_version = '1.3.2'
-        compose_material_version = '1.4.0'
-        dagger_version = "2.43.2"
+        compose_compiler_version = '1.5.12'
```

Comment:
compose kotlin compiler accoding to [this table](https://developer.android.com/jetpack/androidx/releases/compose-kotlin) based on kotlin version